app/reducers/index.js

```diff
@@ -10,16 +10,45 @@ const defaultState = {
       created: 1488984810654,
       avatar: 'sprout',
       status: 0,
+      currentRating: 7,
       ratings: [],
     },
     {
       id: 1,
       name: 'Drink more champagne',
       created: 1488984810658,
-      avatar: 'sprout',
+      avatar: 'pumpkin',
+      status: 0,
+      currentRating: 8,
+      ratings: [],
+    },
+    {
+      id: 2,
+      name: 'Be cool',
+      created: 1488984810658,
+      avatar: 'pepper',
       status: 0,
+      currentRating: 10,
       ratings: [],
     },
+    // {
```

Comment:
If can live with this, I would like to leave the commented out goals in here. Obviously won't be hardcoding any of the goals into the state in the real version. This makes it easier for me to test what the app looks like with different numbers of goals!